.goreleaser.yml

```diff
@@ -27,12 +27,6 @@ checksum:
   name_template: '{{ .ProjectName }}_checksums.txt'
 archives:
 - name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
-  replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
```

Comment:
Not sure about this part.
**I do not know if this part is needed or not.** 
see https://goreleaser.com/deprecations/#archivesreplacements for how it can be reimplemented

Reply:
I think this is fine. The section was never really necessary in the first place, since the values listed here were actually part of the default anyway. The new default will also cover what we need:

https://github.com/goreleaser/goreleaser/blob/2498ea70290792c2537d98dbbdde0dd5ec07c2d6/internal/pipe/archive/archive.go#L75-L78

which yields:

```
{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ with .Arm }}v{{ . }}{{ end }}{{ with .Mips }}_{{ . }}{{ end }}{{ if not (eq .Amd64 "v1") }}{{ .Amd64 }}{{ end }}
```

vs current:

```
'{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
```

I would say keep it as is or just remove the template and use the default.